cmd/influx/config/config.go

```diff
@@ -16,9 +16,10 @@ import (
 type Config struct {
 	Host string `toml:"url" json:"url"`
 	// Token is base64 encoded sequence.
-	Token  string `toml:"token" json:"token"`
-	Org    string `toml:"org" json:"org"`
-	Active bool   `toml:"active" json:"active"`
+	Token          string `toml:"token" json:"token"`
```

Comment:
let's add `name` in here as well

Reply:
I actually tried about adding name today. But, I find out still have to check the uniqueness among the configs, and more magics had to done on the toml side. It's easier and cleaner to do a simple map.

Reply:
you can have both a map for uniqueness and the name in the config. I'm advocating you do both at this point.

Reply:
the map is an internal implementation detail for the config pkg here to manage imo, no reason the cli has to know about how that uniqueness is done